automl/CHANGELOG.md

```diff
@@ -4,6 +4,19 @@
 
 [1]: https://pypi.org/project/google-cloud-automl/#history
 
+## 0.6.0
+
+09-30-2019 10:40 PDT
+
+### New Features
+- add 'image_classification_model_deployment_metadata' arg to 'AutoMlClient.deploy_model' (via synth) ([#9291](https://github.com/googleapis/google-cloud-python/pull/9291))
```

Comment:
AFAIK the release notes should contain full sentences, capitalized and ending with a period?

If so, please adjust that.